### PR TITLE
Rowley.CrossWorks version 4.8.2

### PR DESCRIPTION
**Pull request by fedeericodl**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/28653)

### CODE REVIEW

File: manifests/r/Rowley/CrossWorks/4.8.2/Rowley.CrossWorks.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Rowley.CrossWorks
+PackageVersion: 4.8.2
```
Comment (ItzLevvie) [inline manifests/r/Rowley/CrossWorks/4.8.2/Rowley.CrossWorks.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 4.8
```

File: manifests/r/Rowley/CrossWorks/4.8.2/Rowley.CrossWorks.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Rowley.CrossWorks
+PackageVersion: 4.8.2
```
Comment (ItzLevvie) [inline manifests/r/Rowley/CrossWorks/4.8.2/Rowley.CrossWorks.installer.yaml:5]:
```suggestion
PackageVersion: 4.8
```

File: manifests/r/Rowley/CrossWorks/4.8.2/Rowley.CrossWorks.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Rowley.CrossWorks
+PackageVersion: 4.8.2
```
Comment (ItzLevvie) [inline manifests/r/Rowley/CrossWorks/4.8.2/Rowley.CrossWorks.yaml:5]:
```suggestion
PackageVersion: 4.8
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
image
image
image

EDIT: **You'll need to update the version folder from 4.8.2 to 4.8 as well.**

**Comment by FedeIlLeone** [comment]
> image
> image
> image
> 
> EDIT: **You'll need to update the version folder from 4.8.2 to 4.8 as well.**

I changed 4.8.2 to 4.8, and the folder name to 4.8 instead of 4.8.2.

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**